Sources/SemanticIndex/SemanticIndexManager.swift

```diff
@@ -722,16 +737,20 @@ package final actor SemanticIndexManager {
     }
   }
 
-  /// Update the index store for the given files, assuming that their targets have already been prepared.
+  /// Update the index store for the given files, assuming that their targets has already been prepared.
```

Comment:
```suggestion
  /// Update the index store for the given files, assuming that their target has already been prepared.
```

But part of some later PR is fine, testing has already run on this one.